src/osdc/Objecter.cc

```
@@ -2705,6 +2705,9 @@ int Objecter::_calc_target(op_target_t *t, epoch_t *last_force_resend,
 	t->sort_bitwise,
 	sort_bitwise,
 	pg_t(prev_seed, pgid.pool(), pgid.preferred()))) {
+    // If the interval has changed, then don't resend for repair read
```

We need to return an error to the callback, right?